pubspec.yaml

```
@@ -115,6 +78,4 @@ flutter:
         - asset: assets/fonts/Pretendard-ExtraBold.otf
           weight: 800
         - asset: assets/fonts/Pretendard-Black.otf
-          weight: 900
-  # For details regarding fonts from package dependencies,
-  # see https://flutter.dev/to/font-from-package
+          weight: 900
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)